.travis.yml

```diff
@@ -20,4 +29,4 @@ before_script:
   - golint ./...
 
 script:
-  - go test -v ./...
+  - go test -v ./...
```

Comment:
I suggest to keep the newline at the end of the file. It is consistent with the go source files and it makes it easier to work with the files on the CLI (e.g. `cat file`), because you will not have the prompt on the same line as the last line of the file.